windowing.py

```diff
@@ -3,19 +3,18 @@
 from skimage.util.shape import view_as_windows
 import statistics
 
-# Input: dict_matrix = dict
-# Input: rows = int (number of rows in a trace)
-# Input: window_size = int (# of samples within one window when creating the windows)
-# Input: stride = int (# of samples between the start of each window)
-#
-# Description: takes dictionary that contains x, y, z, nanos keys and transforms
-#			  this into an np array. Then creates windows from this matrix and returns
-#			  the windows
-#
-# Return: windowed_data = np array (np array of the windowed data)
-
 
 def create_windows_from_dictmatrix(dict_matrix, rows, window_size, stride):
+    # Description: takes dictionary that contains x, y, z, nanos keys and transforms
```

Comment:
I think you don't have access to thy python style guide? Here's a sample of Google style documentation

```
def AwsomeFunction(param1, param2, other_silly_variable=None):
  """Does awsome stuff.

  More detailed explanation of what awesome stuff is and what special cases are introduced by other_silly_variable.

  Args:
    param1: A parameter with a certain type. If numpy array adding dimensions and meaning is helpful.
    param2: Another parameter.
    other_silly_variable: Another optional variable.

  Returns:
    Explanation of the return value.

  Raises:
    IOError: Description of expected errors.
  """
```

In some cases a single line comment is totally sufficient. But using docstrings https://www.python.org/dev/peps/pep-0257/ is very helpful, since it can be automatically parsed into documentation